web-bundle/src/main/java/com/graphhopper/http/GraphHopperManaged.java

```diff
@@ -72,24 +71,23 @@ public GraphHopperManaged(GraphHopperConfig configuration, ObjectMapper objectMa
             graphHopper = new GraphHopper(landmarkSplittingFeatureCollection);
         }
         if (!configuration.getString("spatial_rules.location", "").isEmpty()) {
-            throw new RuntimeException("spatial_rules.location has been deprecated. Please use spatial_rules.borders_directory instead.");
+            throw new RuntimeException("spatial_rules.location has been deprecated. Please use custom_area_files instead.");
         }
-        String spatialRuleBordersDirLocation = configuration.getString("spatial_rules.borders_directory", "");
-        if (!spatialRuleBordersDirLocation.isEmpty()) {
-            final Envelope maxBounds = BBox.toEnvelope(BBox.parseBBoxString(configuration.getString("spatial_rules.max_bbox", "-180, 180, -90, 90")));
-            final Path bordersDirectory = Paths.get(spatialRuleBordersDirLocation);
-            List<JsonFeatureCollection> jsonFeatureCollections = new ArrayList<>();
-            try (DirectoryStream<Path> stream = Files.newDirectoryStream(bordersDirectory, "*.{geojson,json}")) {
-                for (Path borderFile : stream) {
-                    try (BufferedReader reader = Files.newBufferedReader(borderFile, StandardCharsets.UTF_8)) {
-                        JsonFeatureCollection jsonFeatureCollection = localObjectMapper.readValue(reader, JsonFeatureCollection.class);
-                        jsonFeatureCollections.add(jsonFeatureCollection);
-                    }
-                }
+        if (!configuration.getString("spatial_rules.borders_directory", "").isEmpty()) {
+            throw new RuntimeException("spatial_rules.borders_directory has been deprecated. Please use custom_area_files instead.");
```

Comment:
Can you additionally point to this PR or to some documentation on how to fix it?